esp-hal/CHANGELOG.md

```diff
@@ -10,15 +10,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - ESP32-S3: Added SDMMC signals (#2556)
-- `dma::{Channel, ChannelRx, ChannelTx}::set_priority` for GDMA devices (#2403)
+- Aded `set_priority` to the `DmaChannel` trait on GDMA devices (#2403, #2526)
```

Comment:
```suggestion
- Added `set_priority` to the `DmaChannel` trait on GDMA devices (#2403, #2526)
```

Reply:
(In the future) For completeness should `set_priority` take an rx and tx priority?

Reply:
No, but rx and tx may have set_priority I guess.